pkgs/os-specific/solo5/default.nix

```diff
@@ -47,6 +48,14 @@ in stdenv.mkDerivation {
       [ -n "$CONFIG_${lib.toUpper bind}" ] && make install-opam-${bind}
     '') targets}
 
+    substituteInPlace $out/bin/solo5-virtio-mkimage \
+      --replace "/usr/lib/syslinux" "${syslinux}/share/syslinux" \
+      --replace "/usr/share/syslinux" "${syslinux}/share/syslinux" \
+      --replace "cp " "cp --no-preserve=mode "
+
+    wrapProgram $out/bin/solo5-virtio-mkimage \
+      --prefix PATH : ${lib.makeBinPath [ dosfstools mtools parted syslinux ]}
```

Comment:
I guess coreutils as well? In theory docker as well, but probably not a good idea to add to the closure always.

Reply:
I've noticed some stuff in nixpkgs breaks with busybox/toybox, but until we have a policy on this I'd rather avoid having multiple coreutils builds sitting on my machine.